extensions/amp-mraid/0.1/amp-mraid.js

```diff
@@ -147,9 +154,8 @@ export class MraidInitializer {
    */
   mraidLoadSuccess_() {
     const mraid = window['mraid'];
-    if (!mraid || !mraid.getState || !mraid.addEventListener
-        || !mraid.close || !mraid.open || !mraid.expand) {
-      this.handleError_(HostServiceError.UNSUPPORTED);
+    if (!mraid || !mraid.getState) {
```

Comment:
Why are you removing some but not all completeness checks?

Reply:
I move the remaining checks to reject individual services

Reply:
I see, makes sense!